src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx

```diff
@@ -480,18 +486,38 @@ function ReportActionCompose({
                                         onValueChange={onValueChange}
                                         didHideComposerInput={didHideComposerInput}
                                     />
-                                    <ReportDropUI
-                                        onDrop={(event: DragEvent) => {
-                                            if (isAttachmentPreviewActive) {
-                                                return;
-                                            }
-                                            const data = event.dataTransfer?.files[0];
-                                            if (data) {
-                                                data.uri = URL.createObjectURL(data);
-                                                displayFileInModal(data);
-                                            }
-                                        }}
-                                    />
+                                    {/* TODO: remove canUseMultiFilesDragAndDrop check after the feature is enabled */}
+                                    {canUseMultiFilesDragAndDrop ? (
+                                        <DropZoneUI
+                                            onDrop={(event: DragEvent) => {
+                                                if (isAttachmentPreviewActive) {
+                                                    return;
+                                                }
+                                                const data = event.dataTransfer?.files[0];
+                                                if (data) {
+                                                    data.uri = URL.createObjectURL(data);
+                                                    displayFileInModal(data);
+                                                }
```

Comment:
Could we make this part into a reusable method to avoid code duplication just below?